src/SFA.DAS.AANHub.Domain/Interfaces/Repositories/IAuditWriteRepository.cs

```diff
@@ -5,5 +5,6 @@ namespace SFA.DAS.AANHub.Domain.Interfaces.Repositories
     public interface IAuditWriteRepository
     {
         void Create(Audit audit);
+
```

Comment:
Why this white space changes?

Reply:
Removed the space. Hopefully will get ignored in merge into main.
I had introduced an Update verb in an earlier commit which was reverted after we re-worked the Patch within the repo.